### PR TITLE
Sidebar changes

### PR DESCRIPTION
**Pull request by Boburm1rzo**
-I have selected the icons that match
-I disabled the function of opening and closing the sidebar -I changed the colors
-Updated css style
-The settings and log out parts have been added to the sidebar
          
the state before any element is selected

image

the state after the sidebar element is selected

image

### CODE REVIEW

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml [hunk 1]
```diff
@@ -71,32 +88,13 @@
              target=".dock-target" enableGestures="false">
     <e-content-template>
```
Comment (Ramadan1011) [inline CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml:90]: I think, it would be better to remove the empty line

**Comment by Ramadan1011** [comment]
Would it be possible to change the color and fontweight of "ishchilar, tekshiruvlar" and "umumiy"?  Because they don't click?

**Comment by jamshidbek7375** [comment]
It turned out well. I like it

**Review by jamshidbek7375 — APPROVED** [review]

**Comment by Mirazyzz** [comment]
Let's give more space between separator texts

**Comment by Mirazyzz** [comment]
are there any other color options we have? Can you upload several options with color so that we could pick one of them?

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml [hunk 2]
```diff
@@ -30,39 +30,56 @@
...
+        text = "Sozlamalar",
+        iconCss = "fa-solid fa-gear",
+        url = "/#"
+    });
+    menuItems.Add(new
+    {
+        text = "Tizimdan chiqish",
+        iconCss = "fa-solid fa-right-from-bracket",
+        url = "/#"
+    });
```
Comment (Mirazyzz) [inline CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml:82]: I think we decided to remove them.

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml [hunk 3]
```diff
@@ -71,32 +88,13 @@
...
-            <i id="sidebar-toggler" class="fa-solid"></i>
-        </div>
-
-        <div class="profile-wrapper">
-            <i class="fa-solid fa-user"></i>
-            <div class="titles d-flex flex-column ps-3">
-                <h6 class="mb-0">@TempData["UserName"]</h6>
-            </div>
+        <div class="app-logo">
+            <img src='~/LogoCheakDrive.png' style="height: 80px; width:80px" />
```
Comment (Mirazyzz) [inline CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml:92]: can we move this styling to **style.css**?

**Review by Mirazyzz — CHANGES_REQUESTED** [review]
Please, check some other options for colors to make visuals even better. Maybe you can provide ChatGPT the background color and theme (white) which we're using and he can suggest some for us?

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml (hunk 2 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml:82]: fixed

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml (hunk 3 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/Views/Shared/_SideBar.cshtml]: fixed

**Review by Mirazyzz — APPROVED** [review]
Looks good

**Review by Ramadan1011 — APPROVED** [review]
Nice. Keep going, bro

**Review by SharifovDeveloper — APPROVED** [review]

**Pull request merged**